src/styles/card.css

```diff
@@ -0,0 +1,84 @@
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  background-color: var(--color-white);
+  border: 1px solid var(--color-white-border);
+  border-radius: 5px;
+}
```

Comment:
Try to arrange the order of styles beautifully
https://codeguide.academy/html-css.html#css-order

Reply:
@serentinos  Hmm, I think that it was ordered already. Lines 2 - 9 block-model, then design

Reply:
you can add an empty line between your properties for a better look
For example:
```
card {
  display: flex;
  box-sizing: border-box;
  flex-direction: column;
  align-items: center;
  justify-content: space-between;
  width: 200px;
  height: 408px;
  padding: 32px 16px 16px;
  
  background-color: var(--color-white);
  border: 1px solid var(--color-white-border);
  border-radius: 5px;
}
```